python/BioSimSpace/FreeEnergy/_relative.py

```diff
@@ -191,6 +216,21 @@ def __init__(self, system, protocol=None, work_dir=None, engine=None,
                     raise NotImplementedError("GROMACS currently only supports the 'full' perturbation "
                                               "type. Please use engine='SOMD' when running multistep "
                                               "perturbation types.")
+                self._exe = _gmx_exe
+            
+            elif engine == "AMBER":
+                _warnings.warn(
+                    "In this branch, AMBER is currently only available for analysis of AMBER output.")                
```

Comment:
I think I initially put this in so when the analysis is merged the amber functionality is present for later on, but as the actual running of Amber simulations for relative isn't yet in the devel it would throw a warning if setting up and running those is tried.

Reply:
Ah, I see, that makes sense. Sorry for misinterpreting it. Yes, please leave this there for now.